cfpb/cfgov-refresh
# Update 404 and 500 error pages

Text & graphic updates to 404 and 500 error pages

## Additions

- New graphic for 500 error page
- Ability to view error pages in dev env

## Changes

- Text on 404 and 500 pages
- Background image placement on 404 and 500 pages

## Testing

- Check out branch & run frontend.sh
- Verify changes on [http://localhost:8000/500/](http://localhost:8000/500/) and [http://localhost:8000/404/](http://localhost:8000/404/)

## Review

- @cfpb/cfgov-frontends 
- @nataliafitzgerald 

## Screenshots

500 page:

1290:

<img width="1279" alt="screen shot 2016-09-13 at 8 45 33 am" src="https://cloud.githubusercontent.com/assets/778171/18481247/67931aee-7990-11e6-8391-f2265138622d.png">

800px:
<img width="787" alt="screen shot 2016-09-13 at 8 30 23 am" src="https://cloud.githubusercontent.com/assets/778171/18481256/6f7dac60-7990-11e6-902d-36f435c8bb87.png">

675px:
<img width="665" alt="screen shot 2016-09-13 at 8 46 33 am" src="https://cloud.githubusercontent.com/assets/778171/18481892/e2cba58a-7992-11e6-99d6-d20a5a4b8e68.png">

600px:
<img width="587" alt="screen shot 2016-09-13 at 8 46 02 am" src="https://cloud.githubusercontent.com/assets/778171/18481906/f0c15996-7992-11e6-81ce-fbda50b8859b.png">

400px:
<img width="384" alt="screen shot 2016-09-13 at 8 46 09 am" src="https://cloud.githubusercontent.com/assets/778171/18481871/c501924e-7992-11e6-8291-88a0954b74f9.png">

404 page:

1290px:
<img width="1272" alt="screen shot 2016-09-13 at 8 41 53 am" src="https://cloud.githubusercontent.com/assets/778171/18481987/482b33e6-7993-11e6-955a-ae23281f8c4c.png">

800px:
<img width="786" alt="screen shot 2016-09-13 at 8 41 26 am" src="https://cloud.githubusercontent.com/assets/778171/18481995/4eaa29b6-7993-11e6-9d65-44f4aa90ea36.png">

675px:
<img width="654" alt="screen shot 2016-09-13 at 8 42 38 am" src="https://cloud.githubusercontent.com/assets/778171/18482003/55080490-7993-11e6-8b9d-7abc3eaa49c0.png">

600px:
<img width="591" alt="screen shot 2016-09-13 at 8 42 14 am" src="https://cloud.githubusercontent.com/assets/778171/18482008/5a19c0b8-7993-11e6-9a2c-579bb32e702d.png">

400px:

<img width="384" alt="screen shot 2016-09-13 at 8 45 14 am" src="https://cloud.githubusercontent.com/assets/778171/18482020/5ebd08be-7993-11e6-8898-86b68e0de249.png">


## Notes

- The image sizing/placement between 600px & 675px is a bit wonky


## Checklist
